### ISSUE TITLE
Add artificial delay for chat on VODs

### ISSUE THREAD

**Issue by jkhazi** [FrankerFaceZ/FrankerFaceZ#955]
**Is your feature request related to a problem? Please describe.**
Viewer messages are out of sync with the VOD.

**Describe the solution you'd like**
I want to be able to input a number(preferably in miliseconds) which will control when new chat messages appear on chat.

**Describe alternatives you've considered**
Decreasing the message buffer and scrolling up, forcing new messages to load off screen and be seen as more messages are posted, bumping them up. With this method to read a long message you have to pause the vod and possibly draw back the VOD instead of just scrolling the chat up to pause it momentarily. Also if the chat slows down or goes faster this doesn't work.

**Comment by VirtusB** [comment]
+1

**Comment by ujjukumar** [comment]
Yeah, please look into this. It will be very useful feature for Vods.
